### PR TITLE
Validation is done before tansa check for ntb

### PR DESCRIPTION
**Pull request by Aca-jov**
SDNTB-256

### CODE REVIEW

**Comment by pavlovicnemanja** [comment]
:+1:

File: scripts/superdesk-authoring/authoring.js [hunk 1]
```diff
@@ -384,6 +384,17 @@ import 'angular-history/history.js';
             });
         };
 
+        this.validateBeforeTansa = function (orig, diff) {
+            diff = extendItem({}, diff);
+
+            this.cleanUpdatesBeforePublishing(orig, diff);
+
+            return api.save('validate', {'act': 'publish', 'type': orig.type, 'validate': diff})
+            .then(function(result) {
```
Comment (gbbr) [inline scripts/superdesk-authoring/authoring.js]: This `.then` statement is redundant, you can remove it. The same result will be returned.

File: scripts/superdesk-authoring/authoring.js [hunk 2]
```diff
@@ -384,6 +384,17 @@ import 'angular-history/history.js';
             });
         };
 
+        this.validateBeforeTansa = function (orig, diff) {
```
Comment (gbbr) [inline scripts/superdesk-authoring/authoring.js]: This function, on its own, seems unrelated to tansa. Would it not be better suited to call it simply `validate` ? Correct me if I'm wrong.

Comment (gbbr) [inline scripts/superdesk-authoring/authoring.js]: @Aca-jov replied on Slack: the `validate` API endpoint is apparently tansa specific and we already have a private validate function, so we can keep as is.

**Comment by gbbr** [comment]
LGTM
